ci/jenkins/test-vmc.sh

```diff
@@ -301,6 +301,19 @@ function copy_image {
   ${SSH_WITH_ANTREA_CI_KEY} -n capv@${IP} "sudo crictl images | grep '<none>' | awk '{print \$3}' | xargs -r crictl rmi"
 }
 
+function copy_test_image {
+  image=$1
+  IP=$2
+  tag=$3
+
+  docker pull ${image} && docker save -o image.tar ${image}
```

Comment:
After copy to capv nodes , need to clean up local tar file i.e image.tar , also can inspect the image if we want to avoid same image pull again !

Reply:
Updated the implementation to pull the same image only once.